### PR TITLE
Update Chinese documents

### PR DESCRIPTION
**Pull request by kvartet**
Crowdin is always in sync with `master`, all changes in english docs submitted to the `v2.0` branch are not included in this update.

### CODE REVIEW

**Review by ultmaster — APPROVED** [review]

File: examples/nas/cream/Cream_zh_CN.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[文档](https://nni.readthedocs.io/en/latest/NAS/Cream.html)
```
Comment (squirrelsc) [inline examples/nas/cream/Cream_zh_CN.md:1]: The `/en` in readthedocs should be replaced.

File: docs/zh_CN/TrainingService/AMLMode.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
+**在 Azure Machine Learning 上运行 Experiment**
+===================================================
+
+NNI 支持在 `AML <https://azure.microsoft.com/en-us/services/machine-learning/>`__ 上运行 Experiment，称为 aml 模式。
```
Comment (squirrelsc) [inline docs/zh_CN/TrainingService/AMLMode.rst:4]: most azure documents have Chinese version, we can provide them.

File: docs/zh_CN/Release.rst [hunk 3]
```diff
@@ -0,0 +1,1120 @@
...
+
+
+* 支持配置 `log 目录 <https://github.com/microsoft/nni/blob/v0.5.1/docs/ExperimentConfig.md>`__ 
+* 支持 `不同级别的日志 <https://github.com/microsoft/nni/blob/v0.5.1/docs/ExperimentConfig.md>`__，使其更易于调试。
+
+文档
+^^^^^^^^^^^^^
+
+
+* 重新组织文档，新的主页位置：https://nni.readthedocs.io/en/latest/
```
Comment (squirrelsc) [inline docs/zh_CN/Release.rst:862]: use `https://nni.readthedocs.io/zh/latest/`

File: docs/zh_CN/TrainingService/KubeflowMode.rst [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+------
+
+
+.. image:: ../../img/kubeflow_training_design.png
+   :target: ../../img/kubeflow_training_design.png
+   :alt: 
+
+Kubeflow 训练平台会实例化一个 Kubernetes 客户端来与 Kubernetes 集群的 API 服务器交互。
+
+对于每个 Trial，会上传本机 codeDir 路径（在 nni_config.yml 中配置）中的所有文件，包括 parameter.cfg 这样的生成的文件到存储卷中。 当前支持两种存储卷：`nfs <https://en.wikipedia.org/wiki/Network_File_System>`__ 和 `azure file storage <https://azure.microsoft.com/en-us/services/storage/files/>`__，需要在 NNI 的 YAML 文件中进行配置。 当文件准备好后，Kubeflow 训练平台会调用 Kubernetes 的 API 来创建 Kubeflow 作业 (\ `tf-operator <https://github.com/kubeflow/tf-operator>`__ 作业或 `pytorch-operator <https://github.com/kubeflow/pytorch-operator>`__ 作业) ，并将存储卷挂载到作业的 pod 中。 Kubeflow 作业的输出文件，例如 stdout, stderr, trial.log 以及模型文件，也会被复制回存储卷。 NNI 会在网页中显示每个 Trial 的存储卷的 URL，以便浏览日志和输出文件。
```
Comment (squirrelsc) [inline docs/zh_CN/TrainingService/KubeflowMode.rst:45]: see if there is good Chinese document on wiki

File: docs/zh_CN/_templates/index.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
 <div>
   <div class="chinese"><a href="https://nni.readthedocs.io/en/latest/">English</a></div>
```
Comment (squirrelsc) [inline docs/zh_CN/_templates/index.html:8]: use zh link

File: CONTRIBUTING_zh_CN.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+该项目采用了 [ Microsoft 开源行为准则 ](https://opensource.microsoft.com/codeofconduct/)。 有关详细信息，请参阅[行为守则常见问题解答](https://opensource.microsoft.com/codeofconduct/faq/)或联系 opencode@microsoft.com 咨询问题或评论。
+
+# 参与贡献
+
+熟悉贡献协议后，即可按照 NNI 开发人员教程，创建第一个 PR =)：
+
+* 推荐新贡献者先从简单的问题开始：['good first issue'](https://github.com/Microsoft/nni/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) 或 ['help-wanted'](https://github.com/microsoft/nni/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)。
+* [NNI 开发环境安装教程](docs/zh_CN/Tutorial/SetupNniDeveloperEnvironment.md)
+* [如何调试](docs/zh_CN/Tutorial/HowToDebug.md)
+* 如果有使用上的问题，可先查看[常见问题解答](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/FAQ.md)。如果没能解决问题，可通过 [Gitter](https://gitter.im/Microsoft/nni?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 联系 NNI 开发团队或在 GitHub 上 [报告问题](https://github.com/microsoft/nni/issues/new/choose)。
```
Comment (squirrelsc) [inline CONTRIBUTING_zh_CN.md:57]: use zh_CN

File: docs/zh_CN/TrainingService/AMLMode.rst (hunk 2 above)
Comment (kvartet) [reply docs/zh_CN/TrainingService/AMLMode.rst]: updated all.

File: docs/zh_CN/Release.rst (hunk 3 above)
Comment (kvartet) [reply docs/zh_CN/Release.rst]: updated it.

File: docs/zh_CN/TrainingService/KubeflowMode.rst (hunk 4 above)
Comment (kvartet) [reply docs/zh_CN/TrainingService/KubeflowMode.rst]: updated all.

File: docs/zh_CN/_templates/index.html (hunk 5 above)
Comment (kvartet) [reply docs/zh_CN/_templates/index.html:8]: This link is for connecting English documents, so it should be `en`.

File: examples/nas/cream/Cream_zh_CN.md (hunk 1 above)
Comment (kvartet) [reply examples/nas/cream/Cream_zh_CN.md]: I forgot the links outside the folder `zh_CN`... it has been updated.

File: CONTRIBUTING_zh_CN.md (hunk 6 above)
Comment (kvartet) [reply CONTRIBUTING_zh_CN.md]: Updated it.

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**